### ISSUE TITLE
There's no way to allow two (or more) 'valid' SIMs - if you live in two countries and alternate between SIMs...

### ISSUE THREAD

**Issue by ghost** [prey/prey-android-client#33]
I live in two countries, and often travel between the two. I have two mobile numbers (...two sim cards), one for each country so that I don't waste money ramping up roaming charges (especially when one wants to use data).

I installed prey on my android, and every time I go to the second country and put the other sim in, it unnecessarily thinks that it's been stolen and goes into 'missing' mode.

I have unticked "Warn on SIM change" in the android app settings for now, but this then turns off that useful feature (when it really happens).

Can there be added a way in the app or online account, to allow x amount of specified SIM cards to all be marked as 'valid' (and not just the current one)?

Thanks.

**Comment by fanuneza** [comment]
Hi, guys.

Multiple-SIM support is not a priority now. We also removed the SMS alter from Prey for Android, so you shouldn't have problems with the feature. You'll get an email alert, but only once.
